### PR TITLE
248: Add loading check method

### PR DESCRIPTION
**Pull request by susannahmarcus**
- tested on first queries test
- gitIgnore node modules and yarn.lock
- add loading check method to error check method
- prettier ran
- changes the config to add more options

### CODE REVIEW

**Review by joshelser — DISMISSED** [review]
LGTM. Should we wait for Vicky to also review?

File: cypress/support/utils.js
Comment (vqmarkman) [inline cypress/support/utils.js:124]: @susannahmarcus, this one should be replaced back to be `type('{command+enter}')`.
Broke tests for me in the past.

Comment (susannahmarcus) [reply cypress/support/utils.js:124]: To the best of my knowledge, this should be fine in that formatting since it just spaced it over two lines, but from a JS perspective, this is the same thing

Comment (susannahmarcus) [reply cypress/support/utils.js:124]: that said, I need to update something that I'm seeing is failing on deploy and see if that helps...

**Comment by susannahmarcus** [comment]
I added something to the clickCheck function which should make it work, despite rerenders (which seems to be a problem in the probes test, so this would help that function in that instance as well)

File: cypress/support/utils.js
Comment (vqmarkman) [reply cypress/support/utils.js:124]: this is my bad, I thought it replaced **command+enter** with **command**
we are good! Sorry about that.

**Review by vqmarkman — APPROVED** [review]
Ship it. Thank you, Susannah!

**Comment by vqmarkman** [comment]
I don't like probes test failure - it is very consistent which indicates we have a problem.
Lemme look at it first before you merge.

**Pull request merged**
